src/backend/aspen/api/conftest.py

```diff
@@ -105,6 +105,7 @@ async def override_get_auth_user(
     if not found_auth_user:
         raise ex.UnauthorizedException("invalid user")
     request.state.auth_user = found_auth_user
+    return found_auth_user
```

Comment:
This was just an oversight in the overridden method - the real method returns this value already.